Tasks/AppCenterDistributeV3/Tests/L0SymMultipleDSYMs_flat_1.ts

```diff
@@ -181,27 +181,12 @@ fs.readdirSync = (folder: string) => {
 
 fs.statSync = (s: string) => {
     let stat = new Stats;
-
-    stat.isFile = () => {
```

Comment:
These are good additions, but technically could be part of another PR?

Reply:
Reverted changes for this file. It will be moved to separated PR